pkgs/development/python-modules/pytorch/default.nix

```diff
@@ -72,6 +67,12 @@ in buildPythonPackage rec {
   PYTORCH_BUILD_VERSION = version;
   PYTORCH_BUILD_NUMBER = 0;
 
+  USE_FBGEMM = 0; # this can't build because of CMAKE downloads
```

Comment:
Worth reviewing the instructions at https://github.com/pytorch/FBGEMM. Looks like we can avoid downloads by setting ASMJIT_SRC_DIR, CPUINFO_SRC_DIR, GOOGLETEST_SOURCE_DIR.

Reply:
So I did set these as environment variables, using pointers from your gist: https://gist.github.com/tbenst/15495f97ffe2a71dc18c49e247af8f51#file-pytorch_11-nix-L62

Unfortunately, these didn't seem to propagate through to CMake. I'll do a round-two later.